src/nidmm/metadata/enums_addon.py

```diff
@@ -17,3 +17,19 @@
     'OffsetCompensatedOhms': {},  # Delete because boolean values only
 }
 
+enums_override_values = {
+    'Function': { 'values': {  # Write out number
```

Comment:
`# Override names that can't be directly converted from C names into valid Python names`

Reply:
Comments changed